core/storage/blog/gae_models.py

```diff
@@ -334,16 +334,25 @@ def has_reference_to_user_id(cls, user_id):
             cls.editor_ids == user_id).get(keys_only=True) is not None
 
     @classmethod
-    def get_by_user(cls, user_id):
+    def get_by_user(cls, user_id, published=False, limit=None):
         """Retrieves the rights object for all blog posts assigned to given user
 
         Args:
             user_id: str. ID of user.
+            published: bool. True if rights model for published blog post are to
+                be fetched else false.
+            limit: int. Number of BlogPostRightsModel to be fetched. All
+                existing models will be fetched if none.
```

Comment:
If None, all existing models will be fetched.

Reply:
Done.